src/server/qgsserver.cpp

```diff
@@ -387,109 +387,121 @@ void QgsServer::handleRequest( QgsServerRequest& request, QgsServerResponse& res
   // Call  requestReady() method (if enabled)
   theResponse.start();
 
-  QMap<QString, QString> parameterMap = request.parameters();
-  printRequestParameters( parameterMap, logLevel );
+  // Plugins may have set exceptions
+  if ( !theRequestHandler.exceptionRaised() )
+  {
+    try
+    {
+      QMap<QString, QString> parameterMap = request.parameters();
+      printRequestParameters( parameterMap, logLevel );
 
-  const QgsAccessControl* accessControl = sServerInterface->accessControls();
+      const QgsAccessControl* accessControl = sServerInterface->accessControls();
 
-  //Config file path
-  QString configFilePath = configPath( *sConfigFilePath, parameterMap );
+      //Config file path
+      QString configFilePath = configPath( *sConfigFilePath, parameterMap );
 
 #ifdef HAVE_SERVER_PYTHON_PLUGINS
-  // XXX Why this is enabled only fol plugins ?
-  sServerInterface->setConfigFilePath( configFilePath );
+      // XXX Why this is enabled only for plugins ?
```

Comment:
@dmarteau that make me think: you should make sure that the server can still be built with WITH_SERVER_PLUGINS disabled.